meganav.yaml

```
@@ -34,5 +724,175 @@ get_ubuntu:
               url: /tutorials/install-ubuntu-desktop#1-overview
             - title: Try before installation
               url: /tutorials/try-ubuntu-before-you-install#1-getting-started
+            - title: Case studies, whitepapers & webinars
+              url: /engage
             - title: All tutorials
               url: /tutorials
+    - title: Server
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Server
+              description: Ubuntu optimsed for servers
+              url: /server
+            - title: MAAS
+              description: Bare metal provisioning
+              url: https://maas.io/
+            - title: Certified hardware
+              description: Machines you can trust
+              url: /certified/servers
+            - title: Public clouds
+              description: Server on AWS, Azure and GCP
```

```suggestion
              description: Servers on AWS, Azure and GCP
```
maybe this is better?

---

Ubuntu Server is a product, let's stick with singular